parsec/backend/organization.py

```diff
@@ -124,9 +126,36 @@ async def api_organization_status(self, client_ctx, msg):
             }
         )
 
-    @api("organization_stats", handshake_types=[APIV1_HandshakeType.ADMINISTRATION])
+    @api("organization_stats", handshake_types=[HandshakeType.AUTHENTICATED])
     @catch_protocol_errors
     async def api_organization_stats(self, client_ctx, msg):
+        msg = organization_stats_serializer.req_load(msg)
+
+        try:
+            if client_ctx.profile != UserProfile.ADMIN:
+                return {
+                    "status": "not_allowed",
+                    "reason": f"User `{client_ctx.device_id.user_id}` is not admin",
+                }
+            # Get organization of the user
+            organization_id = client_ctx.organization_id
+            stats = await self.stats(organization_id)
+
+        except OrganizationNotFoundError:
+            return {"status": "not_found"}
+
+        return organization_stats_serializer.rep_dump(
+            {
+                "status": "ok",
+                "users": stats.users,
+                "data_size": stats.data_size,
+                "metadata_size": stats.metadata_size,
+            }
+        )
+
+    @api("apiv1_organization_stats", handshake_types=[APIV1_HandshakeType.ADMINISTRATION])
+    @catch_protocol_errors
+    async def apiv1_organization_stats(self, client_ctx, msg):
```

Comment:
naming should be `apiv1_administration_organization_stats`

Reply:
done